modules/monitoring-default-monitoring-components.adoc

```diff
@@ -11,40 +11,51 @@ By default, the {product-title} {product-version} monitoring stack includes thes
 [options="header"]
 |===
 
-|Component|Description
+|Component|Description|Version
 
 |Cluster Monitoring Operator
 |The Cluster Monitoring Operator (CMO) is a central component of the monitoring stack. It deploys and manages Prometheus instances, the Thanos Querier, the Telemeter Client, and metrics targets and ensures that they are up to date. The CMO is deployed by the Cluster Version Operator (CVO).
+|v0.1.1
```

Comment:
Similar to telemeter client, this is a downstream component and we should use the OpenShift version?

Reply:
agreed